src/style.css

```
@@ -124,3 +124,6 @@ body {
 .bg-image-5 {
 	background-image: url("image2.jpg");
 }
+
+/* Add Google Font reference */
+@import url('PressStart2P.woff2');
```

Biome (1.9.4)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)